### PR TITLE
Logo added

### LINKED ISSUE #6: Create Logo component
Create Logo component for mobile/tablet+ view that will switch the images depending on the window size.
SVGs for Mobile and Tablet+ are here [Archive.zip](https://github.com/vbetivu-ellation/crunchygram/files/7286202/Archive.zip)

On Tablet/Desktop should be shown **tablet.svg** and on Mobile the **mobile.svg** file

### PR DESCRIPTION
**Pull request by feliciabn**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,30 +1,12 @@
-import logo from "./logo.svg";
-import styles from "./App.module.css";
-import { Text } from "./components/common";
-import StoreProvider from "./store";
-import Counter from "./components/Counter";
+
+import Logo from "./components/common/Logo"
```
Comment (darsenii-crunchyroll) [inline src/App.js]: Please remove its usage from App.js

File: src/App.js [hunk 2]
```diff
@@ -1,30 +1,12 @@
...
-            className={styles.link}
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
-    </StoreProvider>
```
Comment (darsenii-crunchyroll) [inline src/App.js]: Please don't change App.js for now

File: src/components/common/Logo/Logo.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+         display: 'flex',
+         
+      }} >
+        <img className='large' src={tablet} alt="tablet"/>
+        <img className="small" src={mobile} alt="mobile"/>
+      </div>
+    </div>
+  )
+}
+export default Logo;
```
Comment (darsenii-crunchyroll) [inline src/components/common/Logo/Logo.js]: Please always add an empty line in the end of the file

File: src/components/common/Logo/Logo.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  
+  @media (max-width: 568px) {
+    .small {
+      display: block;
+    }
+  
+    .large {
+      display: none;
+    }
+  }
```
Comment (darsenii-crunchyroll) [inline src/components/common/Logo/Logo.css]: Please add an empty line

File: src/components/common/Logo/index.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+import Logo from "./Logo.js"
+export default Logo;
```
Comment (darsenii-crunchyroll) [inline src/components/common/Logo/index.js]: Please add a line between import and export, and another one in the end

File: src/components/common/Logo/Logo.css (hunk 4 above)
Comment (darsenii-crunchyroll) [reply src/components/common/Logo/Logo.css]: Also please rename the file to Logo.module.css

File: src/components/common/Logo/Logo.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import mobile from "./mobile.svg";
+import tablet from './tablet.svg';
+import "./Logo.css";
+
+const Logo = () => {
+  return (
+    <div>
+      <div style={{
```
Comment (darsenii-crunchyroll) [inline src/components/common/Logo/Logo.js]: Move all syles in Logo.module.css

File: src/components/common/Logo/Logo.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+         background: "white",
+         borderBottom: "1px solid #808080",
+         height: "60px",
+         width: "100%",
+         display: 'flex',
+         
+      }} >
+        <img className='large' src={tablet} alt="tablet"/>
```
Comment (darsenii-crunchyroll) [inline src/components/common/Logo/Logo.js]: Let's use SVGs here

**Review by darsenii-crunchyroll — APPROVED** [review]

**Review by vbetivu-ellation — APPROVED** [review]

**Pull request merged**